test/thermo/thermoFromYaml.cpp

```diff
@@ -1,474 +1,489 @@
-    EXPECT_NEAR(thermo->density(), 10.1787080, 1e-6);
-    EXPECT_NEAR(thermo->enthalpy_mass(), -15642788.8547624, 1e-4);
-    EXPECT_NEAR(thermo->gibbs_mole(), -313642312.7114608, 1e-4);
-}
-
-TEST(ThermoFromYaml, Lattice)
-{
-    auto thermo = newThermo("thermo-models.yaml", "Li7Si3_and_interstitials");
-
-    // Regression test based on modified version of Li7Si3_ls.xml
-    EXPECT_NEAR(thermo->enthalpy_mass(), -2077955.0584538165, 1e-6);
-    double mu_ref[] = {-4.62717474e+08, -4.64248485e+07, 1.16370186e+05};
-    double vol_ref[] = {0.095564748201438871, 0.2, 0.09557086};
-    vector_fp mu(thermo->nSpecies());
-    vector_fp vol(thermo->nSpecies());
-    thermo->getChemPotentials(mu.data());
-    thermo->getPartialMolarVolumes(vol.data());
-
-    for (size_t k = 0; k < thermo->nSpecies(); k++) {
-        EXPECT_NEAR(mu[k], mu_ref[k], 1e-7*fabs(mu_ref[k]));
-        EXPECT_NEAR(vol[k], vol_ref[k], 1e-7);
-    }
-}
-
-TEST(ThermoFromYaml, Lattice_fromString)
-{
-    // A little different because we can't re-read the input file to get the
-    // phase definition for the neutral phase
-    std::ifstream infile("../data/thermo-models.yaml");
-    std::stringstream buffer;
-    buffer << infile.rdbuf();
-    AnyMap input = AnyMap::fromYamlString(buffer.str());
-    auto thermo = newPhase(
-        input["phases"].getMapWhere("name", "Li7Si3_and_interstitials"),
-        input);
-
-    // Regression test based on modified version of Li7Si3_ls.xml
-    EXPECT_NEAR(thermo->enthalpy_mass(), -2077955.0584538165, 1e-6);
-    double mu_ref[] = {-4.62717474e+08, -4.64248485e+07, 1.16370186e+05};
-    double vol_ref[] = {0.095564748201438871, 0.2, 0.09557086};
-    vector_fp mu(thermo->nSpecies());
-    vector_fp vol(thermo->nSpecies());
-    thermo->getChemPotentials(mu.data());
-    thermo->getPartialMolarVolumes(vol.data());
-
-    for (size_t k = 0; k < thermo->nSpecies(); k++) {
-        EXPECT_NEAR(mu[k], mu_ref[k], 1e-7*fabs(mu_ref[k]));
-        EXPECT_NEAR(vol[k], vol_ref[k], 1e-7);
-    }
-}
-
-TEST(ThermoFromYaml, Metal)
-{
-    auto thermo = newThermo("thermo-models.yaml", "Metal");
-    EXPECT_DOUBLE_EQ(thermo->density(), 9.0);
-    EXPECT_DOUBLE_EQ(thermo->gibbs_mass(), 0.0);
-}
-
-TEST(ThermoFromYaml, BinarySolutionTabulatedThermo)
-{
-    auto thermo = newThermo("thermo-models.yaml", "graphite-anode");
-    EXPECT_NEAR(thermo->density(), 5031.7, 1e-5);
-    EXPECT_NEAR(thermo->enthalpy_mass(), -32501.245047302145, 1e-9);
-    EXPECT_NEAR(thermo->entropy_mass(), 90.443481807823474, 1e-12);
-    thermo->setMoleFractionsByName("Li[anode]: 0.55, V[anode]: 0.45");
-    EXPECT_NEAR(thermo->gibbs_mass(), -87066.246182649265, 1e-9);
-}
-
-TEST(ThermoFromYaml, DeprecatedPhase)
-{
-    // The deprecation warning in this file is turned into an
-    // error by make_deprecation_warnings_fatal() called in main()
-    // for the test suite.
-    EXPECT_THROW(newThermo("gri30.yaml", "gri30_mix"), CanteraError);
-}
+#include "gtest/gtest.h"

```

Comment:
Please make sure that the line endings in this file remain `LF`, you've accidentally changed them to `CRLF`

Reply:
The same thing happened to a few other files, including `MixtureFugacityMFTP.h` and `RedlichKwongMFTP.h` and the associated `.cpp` files.

Reply:
Corrected.

Reply:
Just pushed a commit that should fix this.